test/controllers/conftest.py

```diff
@@ -106,6 +107,7 @@ def expected_report_rows():
             "07000000001",
             "test1@ons.gov.uk",
             "ACTIVE",
+            "2024-01-29 14:16:02.105 +0000",
```

Comment:
Might be worth checking what is expected here with regards to format, I am doubtful the business would want a full timestamp like this

Reply:
This has been reformatted.